packages/react/src/components/tooltip/Tooltip.tsx

```diff
@@ -127,6 +127,7 @@ export const Tooltip = ({
         aria-label={buttonLabel}
         aria-expanded={isTooltipOpen}
         onClick={onButtonClick}
+        onBlur={() => setIsTooltipOpen(false)}
```

Comment:
BTW is there a performance issue when using like this instead of using for example `const onBlurHandler = () => setToolTipOpen(false);` outside the render and calling it like `onBlur={onBlurHandler}`. If I remember correctly, React recreates the functions when rendering if declared like {() => {}}.

Reply:
interesting, could be.